<application>
  <component name="AppStorage">
    <option name="newTranslationDialogX" value="531" />
    <option name="newTranslationDialogY" value="358" />
    <option name="pinTranslationDialog" value="true" />
    <histories>
      <item value="Compliant Solution" />
      <item value="a newline plus indentation" />
      <item value="More precisely" />
      <item value="calc will have no effect" />
      <item value="&quot;calc&quot; operands should be correctly spaced" />
      <item value="correctly" />
      <item value="The W3C specifications define the units that can be used with lengths. A unit that is not part of the list of supported ones is likely to be a typo and will be seen as a UI bug by the user." />
      <item value="specifications" />
      <item value="The W3C specifications define the valid CSS properties. Only the official and browser-specific properties should be used to get the expected impact in the final rendering." />
      <item value="CSS properties should be valid" />
      <item value="The W3C specifications say comments should be defined using ... . The use of is not supported on all browsers and can lead to unexpected results." />
      <item value="Single line comment syntax should not be used" />
      <item value="Noncompliant" />
      <item value="Noncompliant Code Example" />
      <item value="Arrays in JavaScript have several methods for filtering, mapping, or folding that require a callback. Not having a return statement in such a callback function is most likely a mistake, because processing of the array uses the return value of the callback. If there is no return, callback will implicitly return undefined, which will likely fail." />
      <item value="最佳实践" />
      <item value="data transmission" />
      <item value="数据传递" />
      <item value="practice" />
      <item value="best practice" />
      <item value="语种列表" />
      <item value="domain name" />
      <item value="域名" />
      <item value="business directory" />
      <item value="企业名录" />
      <item value="repayment amount" />
      <item value="还款金额" />
      <item value="partial payment" />
      <item value="部分还款" />
      <item value="5、依据监管要求，订单信息将作为人民币结汇的依据，请保证信息的真实性，否则将影响款项到账。" />
      <item value="4、审核拒绝的店铺不支持上传订单；" />
      <item value="3、上传订单的大小请勿超过20 MB，若文件过大请分开上传；" />
      <item value="2、请正确上传对应店铺180天内的订单，超过180天的订单系统将无法识别；" />
      <item value="1、请勿增加删除修改模版中表头字段及顺序；" />
      <item value="上传订单注意事项" />
      <item value="折算后" />
      <item value="可用结汇额度为您当前店铺的原始贸易订单金额。" />
      <item value="店铺审核拒绝，请修改店铺信息后再来上传订单" />
      <item value="优化" />
      <item value="结汇额度优化" />
      <item value="nano" />
      <item value="epoch Second" />
      <item value="差异" />
      <item value="nonsupport" />
      <item value="不支持" />
      <item value="invalidate" />
      <item value="法人名称" />
      <item value="统一社会信用代码" />
      <item value="联行行号" />
      <item value="开户支行" />
    </histories>
    <option name="languageScores">
      <map>
        <entry key="CHINESE" value="423" />
        <entry key="ENGLISH" value="410" />
        <entry key="AFRIKAANS" value="2" />
        <entry key="DANISH" value="1" />
        <entry key="FILIPINO" value="4" />
        <entry key="ROMANIAN" value="1" />
        <entry key="NORWEGIAN" value="1" />
        <entry key="PORTUGUESE" value="2" />
        <entry key="HINDI" value="1" />
        <entry key="INDONESIAN" value="4" />
      </map>
    </option>
  </component>
  <component name="Cache">
    <option name="lastTrimTime" value="1672308140759" />
  </component>
  <component name="Settings">
    <option name="translator" value="YOUDAO" />
    <option name="youdaoTranslateSettings">
      <youdao-translate>
        <option name="appId" value="1e081ca7b68e1943" />
        <option name="primaryLanguage" value="ESPERANTO" />
      </youdao-translate>
    </option>
  </component>
</application>